### PR TITLE
Add Euclidean Algorithm in Emojicode.

### PR DESCRIPTION
**Pull request by julianschacher**

### CODE REVIEW

**Comment by jiegillet** [comment]
Here wo go again ^^

File: contents/euclidean_algorithm/code/emojicode/euclidean_algorithm.emojic [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      var_a 🚮 var_b ➡️ 🖍var_b
+      temp ➡️ 🖍var_a
+    🍉
+
+    ↩️ var_a
+  🍉
+🍉
+
+🏁 🍇
+  😀 🔡 ️🔼🐇⬆️ 🤜64 ✖️ 67🤛 🤜64 ✖️ 81🤛 ❗️ 10 ❗️❗️
```
Comment (thbwd) [inline contents/euclidean_algorithm/code/emojicode/euclidean_algorithm.emojic]: There’s really no need to wrap these arguments into  `🤜🤛`.(Unless you want to do that for readability.)

**Comment by leios** [comment]
@thbwd Thanks again for the review! You rock!

File: contents/euclidean_algorithm/code/emojicode/euclidean_algorithm.emojic (hunk 1 above)
Comment (julianschacher) [reply contents/euclidean_algorithm/code/emojicode/euclidean_algorithm.emojic]: Yup, I think it improves the readability.

**Pull request merged**